hera_mc/cm_hookup.py

```diff
@@ -191,11 +195,12 @@ def __get_hookup(self, hpn_list, rev, port_query, at_date,
         for k, part in parts.iteritems():
             if not cm_utils.is_active(self.at_date, part['part'].start_date, part['part'].stop_date):
                 continue
-            hookup_dict['hookup'][k] = {}
+            huk = k
```

Comment:
Is there a reason to (essentially) rename this variable in this way? From my read of the code it seems as if one could write `for huk, par in parts.iteritems()`. Or just leave it as `k`.

If there's a reason for this that's escaping me, it should be documented in a comment.

Reply:
As soon as this PR is done, I'm going to start a new one where I resolve a worry I have about the hookup key.  I can reset it for now...